test/CoreTypechecker.hs

```diff
@@ -1,142 +1,176 @@
+{-# LANGUAGE ExplicitForAll #-}
```

Comment:
I don't think you need this here either.

Reply:
Sorry I missed this. Fixed.